include/tbl/vsl_tags.h

```diff
@@ -691,6 +691,17 @@ SLTM(VdpAcct, 0, "Deliver filter accounting",
 	NODEF_NOTICE
 )
 
+SLTM(DeliveryError, 0, "Error while delivering an HTTP message.",
+	"Logs the error message of a failed delivery operation.\n\n"
```

Comment:
Maybe add

> (for example client response or backend request)

?

Reply:
A few lines below it tells us that the prefix is either `bereq:` or `resp:`.